db/migrate/20190715090649_add_device_id_indexes.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeviceIdIndexes < ActiveRecord::Migration[5.2]
+  def change
+    add_index :measurements, :device_id
```

Comment:
This index (device_id) is already present in the production database, it was added manually by Mapview, and unfortunately ActiveRecord generates `CREATE INDEX` commands, not `CREATE INDEX IF NOT EXISTS` commands, so I may need to modify this to get it to work in production somehow? I'm not sure. Also need to check if it's already present on the dev database.

Reply:
My approach here would be to make a migration just for the already-created index then manually insert that into the `schema_migrations` on dev & prd so it's as if it already ran.

Then have another migration for the `device_urn` index that we'll run with db:migrate as usual.